salesforce-pubsub/1.0/modules/ROOT/pages/salesforce-pubsub-connector-message-tracking-example.adoc

```diff
@@ -2,7 +2,7 @@
 
 One way to use Salesforce Pub/Sub Connector is to track whether a Salesforce platform event is received from MuleSoft and processed. You can keep track of processed messages using a combination of Salesforce and Mule runtime engine (Mule). 
 
-For a detailed example of one approach to message tracking, see https://docs.google.com/document/d/1NXLE93nzo7IRy4HSHsxh_InhhDJGv_m_xGo91VDyb3U/edit#heading=h.3xat2b92h1uu[Keeping Track of the Processed and Received Messages].
+For a detailed example of one approach to message tracking, see xref:attachment$salesforce-pubsub-whitepaper.pdf[Keeping Track of the Processed and Received Messages].
```

Comment:
Wow, I didn't even know you could do an attachment like this! I learned something new!

Reply:
Yup I checked with Gary and it is possible for the MS docs site!